### PR TITLE
Description update

### LINKED ISSUE #799: Potential character encoding issues in agency descriptions
On this one (https://open.foia.gov/contacts/united-states-international-boundary-and-water-com), there's a character encoding issue in the agency description (the question mark).

### PR DESCRIPTION
**Pull request by geramirez**
Fixes https://github.com/18F/foia-hub/issues/799 and other descriptions.

### CODE REVIEW

**Comment by khandelwal** [comment]
Shouldn't there be a code change associated with this?

**Comment by geramirez** [comment]
@khandelwal Originally, I ran the manual data script to make the changes; however, [this](https://github.com/18F/foia/pull/117/files#diff-309fafdfe61c13813a6af79e4d46fac4R112) change should stop the data manual descriptions from being overwritten.

**Comment by khandelwal** [comment]
Wait, I don't understand. Where the manual changes made elsewhere? If we're having encoding issues with the USA.gov API, shouldn't we solve those first? It doesn't make sense to make manual edits to fix an encoding issue -- unless that's the only way to fix them.

**Comment by geramirez** [comment]
I only made manual changes to the USIBWC; however, while working on this issue also noticed that `layer_with_usa_contacts.py` was overwriting manually added descriptions with data from USA.gov API. In the past we assumed that the USA.gov API had the best descriptions, but after reviewing the data it appears that the descriptions we added manually are better.

File: contacts/layer_with_usa_contacts.py [hunk 1]
```diff
@@ -109,7 +109,7 @@ def update_dict(old_data, new_data):
     """
 
     old_data['usa_id'] = new_data.get('usa_id')
-    if new_data.get('description'):
+    if new_data.get('description') and not old_data.get('description'):
```
Comment (khandelwal) [inline contacts/layer_with_usa_contacts.py:112]: Is this because we're running this after we run a manual update?

**Pull request merged**